### DISCUSSION TITLE
Bring Your Own Starter Kit

### DISCUSSION THREAD

**Discussion by tomgobich** [adonisjs/core#4365] [Help] [answered]
I think there might either 
- Be an issue with using GitHub as the default repository provider with the bring-your-own starter kit 
- There might be an issue within the `giget` package using GitHub as the default
- The documentation might need tweaking
- Or, some combination of the above lol

When run as currently documented in the AdonisJS 6 docs, I'm getting the following error where it attempts to download from a non-existent URL.
<img width="833" alt="Screenshot 2024-01-28 at 8 37 48 AM" src="https://github.com/adonisjs/core/assets/19398726/a27a2471-64ea-4c3f-988a-fdb47e6f14e9">

However, if I prefix it with `github:` all works a-okay
<img width="815" alt="Screenshot 2024-01-28 at 8 37 07 AM" src="https://github.com/adonisjs/core/assets/19398726/2deae73e-495e-4447-9057-f1b1c9e1e6c1">

Looks like `giget` _should_ default to GitHub though...
<img width="876" alt="Screenshot 2024-01-28 at 8 49 03 AM" src="https://github.com/adonisjs/core/assets/19398726/2d28bb23-eb90-4f56-b4c7-4ea93cc626fc">

So, figured I'd create a discussion instead of an issue since I'm really not sure whether it's something in `create-adonisjs`, `giget`, or the docs.

**Comment by Julien-R44** [comment, marked as answer]
hey Tom, thanks! this was was fixed with this commit : https://github.com/adonisjs/create-adonisjs/commit/441fb7238083334522dfc4b070e839b01252c6b4

```
pnpm create adonisjs@latest --kit adocasts/lets-learn-adonisjs-6
```

is now working fine

**Reply by tomgobich** [reply]
Awesome, thanks a ton Julien!! 😄
